src/main/java/frc/robot/BuildConstants.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot;
+
+/**
+ * Automatically generated file containing build version information.
+ */
+public final class BuildConstants {
+  public static final String MAVEN_GROUP = "";
+  public static final String MAVEN_NAME = "Warbots2023";
+  public static final String VERSION = "unspecified";
+  public static final int GIT_REVISION = 92;
+  public static final String GIT_SHA = "5a56bf58e35b7a5ab995af45563eb50686b8e9c8";
+  public static final String GIT_DATE = "2023-02-03 09:33:47 EST";
+  public static final String GIT_BRANCH = "Vision";
+  public static final String BUILD_DATE = "2023-02-06 19:44:45 EST";
+  public static final long BUILD_UNIX_TIME = 1675730685558L;
+  public static final int DIRTY = 1;
+
+  private BuildConstants(){}
+}
```

Comment:
Delete This file. This File should not be committed. This file is a product of building the robot code.